django/thunderstore/api/cyberstorm/views/markdown.py

```diff
@@ -0,0 +1,77 @@
+from typing import Optional
+
+from django.http import Http404
+from rest_framework import serializers
+from rest_framework.generics import RetrieveAPIView, get_object_or_404
+
+from thunderstore.api.utils import CyberstormAutoSchemaMixin
+from thunderstore.markdown.templatetags.markdownify import render_markdown
+from thunderstore.repository.models import Package, PackageVersion
+
+
+class CyberstormMarkdownResponseSerializer(serializers.Serializer):
+    html = serializers.CharField()
+
+
+class PackageVersionReadmeAPIView(CyberstormAutoSchemaMixin, RetrieveAPIView):
+    """
+    Return README.md prerendered as HTML.
+
+    If no version number is provided, the latest version is used.
+    """
+
+    serializer_class = CyberstormMarkdownResponseSerializer
+
+    def get_object(self):
+        package_version = get_package_version(
+            self.kwargs["namespace_id"],
+            self.kwargs["package_name"],
+            self.kwargs.get("version_number"),
+        )
+
+        return {"html": render_markdown(package_version.readme)}
+
+
+class PackageVersionChangelogAPIView(CyberstormAutoSchemaMixin, RetrieveAPIView):
+    """
+    Return CHANGELOG.md prerendered as HTML.
+
+    If no version number is provided, the latest version is used.
+    """
+
+    serializer_class = CyberstormMarkdownResponseSerializer
+
+    def get_object(self):
+        package_version = get_package_version(
+            self.kwargs["namespace_id"],
+            self.kwargs["package_name"],
+            self.kwargs.get("version_number"),
+        )
```

Comment:
https://github.com/thunderstore-io/Thunderstore/pull/918#discussion_r1423673326 applies here and the other view.

Not a blocker

Reply:
Handled in https://github.com/thunderstore-io/Thunderstore/pull/967